src/lib/constants/acm-paths.ts

```diff
@@ -3,41 +3,75 @@ export interface AcmPath {
   slug: string;
   picture: string;
   color: string;
+  blurp: string;
+  blurpRecommend: string;
+  recommendations: string[];
+  workshopPictures: string[];
```

Comment:
Neat idea to add more team-specific constants. I also reworded slightly.
```suggestion
  description: string;
  recommendationsBlurb: string;
  recommendations: Array<{ title: string; link: string; }>;
  workshopPictures: string[];
```

Reply:
Done

Reply:
The word is "blurb", I believe. But `blurp` is fine if you insist.